ci-operator/step-registry/redhat-developer/rhdh/eks/mapt/orphaned/aws-cleanup/redhat-developer-rhdh-eks-mapt-orphaned-aws-cleanup-commands.sh

```
@@ -0,0 +1,385 @@
+        ATTACHMENT=$(aws ec2 describe-network-interfaces --network-interface-ids "${eni}" --query 'NetworkInterfaces[0].Attachment.AttachmentId' --output text 2>/dev/null || true)
+        if [[ -n "${ATTACHMENT}" ]] && [[ "${ATTACHMENT}" != "None" ]]; then
+          aws ec2 detach-network-interface --attachment-id "${ATTACHMENT}" --force || true
+          sleep 5 # Wait a bit after detaching
+        fi
+        aws ec2 delete-network-interface --network-interface-id "${eni}" || true
+      fi
+    done
+    
+    # Security Groups
+    local SGS
+    SGS=$(aws ec2 describe-security-groups --filters Name=vpc-id,Values="${VPC_ID}" --query 'SecurityGroups[?GroupName!=`default`].GroupId' --output text 2>/dev/null || true)
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)